wiki/Tournaments/#GO/3/en.md

```diff
@@ -0,0 +1,159 @@
+| Designer | ![][flag_FR] [0xy](/users/9239673) |
+| Streamers | ![][flag_FR] [Sardoche](/users/4383702), ![][flag_FR] [Kaanibaru](/users/3117824), ![][flag_FR] [FulgurentKille](/users/4096322), ![][flag_US] [ITotallyGetThat](/users/8770622), ![][flag_CA] [kymotsujason](/users/2541804), ![][flag_CA] [Sinaeb](/users/1576095), ![][flag_FR] [Kaeldori](/users/962519) |
+| Casters | ![][flag_US] [Apraxia](/users/4194445), ![][flag_FR] [-raizen-](/users/3872987), ![][flag_FR] [0xy](/users/9239673), ![][flag_FR] [Sardoche](/users/4383702), ![][flag_FR] [Kynan](/users/1093361), ![][flag_FR] [Kaanibaru](/users/3117824), ![][flag_FR] [IIFirenII](/users/1984236), ![][flag_FR] [Battquif](/users/2389333), ![][flag_FR] [Elysion](/users/106269), ![][flag_US] [Dohland](/users/5220511), ![][flag_GB] [Doomsday](/users/18983), ![][flag_US] [BeasttrollMC](/users/3171691), ![][flag_GB] [Bubbleman](/users/5182050), ![][flag_US] [fieryrage](/users/3533958), ![][flag_FR] [Jean-Michel Jr](/users/4430948) |
+| Referees | ![][flag_HK] [Akali393394](/users/9686628), ![][flag_FR] [FayeurS 5](/users/6177263), ![][flag_US] [tigereyes144](/users/6499811), ![][flag_CA] [Sinaeb](/users/1576095), ![][flag_FR] [Kebab De Poche](/users/6467693), ![][flag_FR] [Mimir](/users/7382734), ![][flag_FR] [Tigzick](/users/6745742), ![][flag_FR] [Tigre de poche](/users/5410201), ![][flag_FR] [Mirthille](/users/7548517), ![][flag_PL] [P a t r i c k](/users/6814521), ![][flag_CH] [Icerite](/users/7226287), ![][flag_PL] [Kondi](/users/7382321), ![][flag_FR] [Ryumi](/users/6596270), ![][flag_US] [Raoul](/users/9924405), ![][flag_US] [Cindelluna](/users/3039847), ![][flag_AU] [Cactie](/users/5592409), ![][flag_US] [R-2E-054](/users/3625265), ![][flag_AU] [PingLaw](/users/5401523), ![][flag_RU] [Xassie](/users/7958911), ![][flag_GB] [Yazzehh](/users/7068973), ![][flag_MA] [clorox_1g](/users/9702728), ![][flag_US] [Tuco](/users/11706350), ![][flag_DE] [NKJ](/users/9675790), ![][flag_DE] [AFKBRB](/users/8449331) |
+| Statistician | ![][flag_FR] [FayeurS 5](/users/6177263) |
+| Developers | ![][flag_FR] [Loazo](/users/3925586), ![][flag_SE] [AntoN](/users/2538562), ![][flag_FR] [0xy](/users/9239673) |
+| Wiki | ![][flag_ID] [Naruk](/users/7100002) |
+
+## Links
+
+- [Discussion thread](/community/forums/topics/869989)
+- [#GO Discord server](https://discord.gg/trnFByC)
+- [#GO Youtube channel](https://www.youtube.com/channel/UCwIJ0ymAahAASp0_KaZ_1jA)
+- [Website](https://www.gameosu.com/)
+- [Player registration form](https://www.gameosu.com/sign_up)
+- Livestream
+  - [#Game\_Osu (EN)](https://www.twitch.tv/gameosu)
+  - [Sardoche's Twitch channel](https://www.twitch.tv/sardoche)
```

Comment:
what is this used for? doesn't seem to be explained anywhere

Reply:
I believe the host told me the Finals and Grand Finals will be streamed on Sardoche's. He's a massive FR Twitch streamer too, that may attract more viewers to the tournament.

Reply:
can write that in parenthesis after this link then